nats-base-client/kv.ts

```diff
@@ -231,6 +234,10 @@ export class Bucket implements KV, KvRemove {
     const discardNew = have ? compare(have, parseSemVer("2.7.2")) >= 0 : false;
     sc.discard = discardNew ? DiscardPolicy.New : DiscardPolicy.Old;
 
+    const direct = have ? compare(have, parseSemVer("2.8.5")) >= 0 : false;
```

Comment:
Seeing that we already have to check server version compatibility twice here, maybe it would be good to add and export a `checkCompat(have, want)` function in `semver.ts`?